tests/nosetests/pyanaconda_tests/simple_ui_test.py

```diff
@@ -70,6 +75,20 @@ def _get_category_names(self, hub_class):
         categories = self._get_categories(hub_class)
         return {c.__name__:  list(sorted(s.__name__ for s in categories[c])) for c in categories}
 
+    def _check_spokes_priority_uniqueness(self, spokes):
+        res = dict()
+
+        for spoke in spokes:
+            priority = spoke.priority
+            name = spoke.__name__
+
+            if priority in res:
+                self.assertNotIn(
+                    priority,
+                    res,
+                    "Spokes {} and {} have the same priority!".format(res[priority], name))
```

Comment:
I would create the message first and then call the assert method, so all arguments can be on the same line.

Reply:
Fixed.